int16 packet_source
int8 packet_version
int8 packet_type
string packet_data
int16 packet_ttl
int64 package_check_sum
